tools/wpt/run.py

```diff
@@ -221,6 +221,9 @@ def setup_kwargs(self, kwargs):
                                                     channel=kwargs["browser_channel"])
             kwargs["prefs_root"] = prefs_root
 
+        # Allow WebRTC tests to call getUserMedia.
+        kwargs["extra_prefs"].append("media.navigator.streams.fake=true")
```

Comment:
Honestly I think this should be in https://searchfox.org/mozilla-central/source/testing/profiles/unittest/user.js or the wpt-specific equivalent that @whimboo is adding. But I guess here is OK temporarily if it's inconvenient to make an upstream patch.

Reply:
I'd be happy to make a patch once @whimboo creates the wpt-specific profile, and I'll remove this afterwards. In the interim, let's land this to make Taskcluster results more useful.

Reply:
Sent a patch to m-c: https://phabricator.services.mozilla.com/D6579